### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by EvaKaul**
- updated Readme page with current infos
- updated Configuration page with current infos and screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,77 +10,96 @@ This example shows how to use the Industrial Edge App "Performance Insight" with
...
+This document describes how to create an OEE analysis dashboard within Performance Insight. The OEE settings are configured based on [this](#tia-project) TIA project and then displayed as a dashboard.
+
+This HowTo is a supplement to [Performance Insight getting started](https://github.com/industrial-edge/performance-insight-getting-started).
+
+![OEEDashboard](/docs/graphics/OEEDashboard.png)
 
 ## Requirements
 
-###  Prerequisities
+### Prerequisities
```
Comment (j0hannesfr) [inline README.md:44]: Prerequisites

File: docs/Installation.md [hunk 2]
```diff
@@ -3,62 +3,53 @@
...
-<p align="center"><kbd><img src="graphics/Performance_Insight_KPI_good_count.PNG" /></kbd></p>
-
-### Configure Parameter
+## Configure Performance Insight
 
-Open the parameters in the respective asset and create two new KPI instances for "theoretical speed" and "good count".
+### Create status mapping
 
-<p align="center"><kbd><img src="graphics/Performance_Insight_parameter_good_count.PNG" /></kbd></p>
+The machine state of a PLC is one of the main parts of the OEE calculation. To correctly map this data, you need to create a status mapping within Performance Insight. All time categories (Net Production Time, Planned Downtime, Unplanned Downtime) must be covered by the mapping. Go to 'Configuration' > 'Status mappings' and create a new status mapping according to this example:
```
Comment (j0hannesfr) [inline docs/Installation.md:26]: Maybe add the OEE_Time.png here?

**Review by j0hannesfr — COMMENTED** [review]
2 small spelling mistakes and it seems like one picture you uploaded is mising in the text. Except of that everything looks good to me

File: README.md [hunk 3]
```diff
@@ -10,77 +10,96 @@ This example shows how to use the Industrial Edge App "Performance Insight" with
...
+  - Common Connector Configurator V 1.8.1-4
+  - Data Service V 1.5.0
+  - Performance Insight V 1.6.2
+- Industrial Edge Device V 1.9.0-27
 - TIA Portal V16
 - CPU 1511-1 PN
 
 ### TIA Project
 
 The used TIA Portal project can be found in the [miscellenous repository](https://github.com/industrial-edge/miscellaneous/tree/main/tank%20application) and is also used for several further application examples.
```
[the remark this answers is not in the document]
Comment (EvaKaul) [reply README.md]: fixed

File: docs/Installation.md (hunk 2 above)
Comment (EvaKaul) [reply docs/Installation.md]: added screenshot

**Comment by EvaKaul** [comment]
Thanks for the findings, Johannes! Please check my fixes and finish the review. Afterwards I can merge the code.

**Review by j0hannesfr — APPROVED** [review]

**Pull request merged**